sdk/avs/Azure.ResourceManager.Avs/src/Customized/AvsPrivateCloudResource.cs

```diff
@@ -0,0 +1,745 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.ResourceManager.Avs.Models;
+using Azure.ResourceManager.Resources;
+
+namespace Azure.ResourceManager.Avs
+{
+    /// <summary>
+    /// A Class representing an AvsPrivateCloud along with the instance operations that can be performed on it.
+    /// If you have a <see cref="ResourceIdentifier"/> you can construct an <see cref="AvsPrivateCloudResource"/>
+    /// from an instance of <see cref="ArmClient"/> using the GetAvsPrivateCloudResource method.
+    /// Otherwise you can get one from its parent resource <see cref="ResourceGroupResource"/> using the GetAvsPrivateCloud method.
+    /// </summary>
+    public partial class AvsPrivateCloudResource : ArmResource
+    {
+        /// <summary>
+        /// Get a private cloud workload network.
+        /// <list type="bullet">
+        /// <item>
+        /// <term>Request Path</term>
+        /// <description>/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AVS/privateClouds/{privateCloudName}/workloadNetworks/{workloadNetworkName}</description>
+        /// </item>
+        /// <item>
+        /// <term>Operation Id</term>
+        /// <description>WorkloadNetworks_Get</description>
+        /// </item>
+        /// <item>
+        /// <term>Default Api Version</term>
+        /// <description>2023-09-01</description>
+        /// </item>
+        /// <item>
+        /// <term>Resource</term>
+        /// <description><see cref="WorkloadNetworkResource"/></description>
+        /// </item>
+        /// </list>
+        /// </summary>
+        /// <param name="workloadNetworkName"> Name for the workload network in the private cloud. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        [ForwardsClientCalls]
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public virtual Response<WorkloadNetworkResource> GetWorkloadNetwork(WorkloadNetworkName workloadNetworkName, CancellationToken cancellationToken = default)
+        {
+            throw new NotSupportedException("This method is obsolete.");
+        }
```

Comment:
will our old api-version still support this operation or all api-versions have been updated to throw service errors on this operation?
If old api-version still supports it, we should not do this by changing its implementation to a throw, because the SDK supports api-version replacement. We will have to copy its old implementation here (and all the related classes and methods)

Reply:
sure, you current implementation also works.
Initially I am suggesting to change it to
```
GetWorkloadNetwork().GetAsync(cancellationToken).ConfigureAwait(false);
```
but yours works as well.